css/fonts.css

```diff
@@ -6,3 +6,21 @@
   font-weight: bold;
   font-style: normal;
 }
+
+@font-face {
+  font-family: 'roboto-medium';
+  src: url('./fonts/roboto-medium-webfont.woff2') format('woff2'),
+       url('./fonts/roboto-medium-webfont.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+
+}
+
+@font-face {
+  font-family: 'roboto-regular';
+  src: url('./fonts/roboto-regular-webfont.woff2') format('woff2'),
+       url('./fonts/roboto-regular-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6313902)